### DISCUSSION TITLE
HyperLogLog (적은 바이트로 대략적인 카디널리티 구하기)

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#374] [1. 개발 관련 글 모음]
실무에서 HyperLogLog를 쓸 일이 있어서 찾아봤다가 원리가 재밌어서 정리해봤습니다~

남들이 잘 안쓰는 주제를 찾다 보니 점점 마이너해지는 것 같아요... 메이저 한 것도 이제 올려보겠습니다 ㅎㅎ

https://gusah009.github.io/etc/2024-03-11-HyperLogLog/

**Comment by jojaeng2** [comment]
벽이 좀 느껴지네요 ..
